### DISCUSSION TITLE
Add debugging mode for Image placeholder

### DISCUSSION THREAD

**Discussion by snelsi** [vercel/next.js#26213] [Ideas]
### Describe the feature you'd like to request

Currently, if you define a blurry placeholder for your image, you can only view it for a brief moment while the image is loading in the background. Maybe, I want to take an artistic look at how my images look or add and tweak effects like [shimmer](https://image-component.nextjs.gallery/shimmer). In these situations, it might be useful to turn on 'debugging mode' and only show a placeholder even when the main image is loaded.

There is no way to programmatically display the Image placeholder, or at least it's not mentioned in the documentation. In localhost with good internet speed, it will be displayed for a split-second. Slow 3g speed in dev tools network doesn't really help. 

### Describe the solution you'd like

Add a boolean prop to always display the placeholder:

```tsx
import src from "public/image.jpg"

<Image
   src={src}
   placeholder="blur"
   showPlaceholder={true} // <- This will make placeholder visible
 />
```

Besides debugging, it might also give developers a little bit more control over how Images are displayed in their app.

### Describe alternatives you've considered

I tried:
- Slowing down my internet speed
- Passing blurDataUrl as a src directly
But it didn't quite work out in the end.

**Comment by timneutkens** [comment]
You can also disable JavaScript on the page temporarily as that will keep the placeholder being shown
